scripts/s2i-generate.sh

```diff
@@ -12,6 +12,7 @@ source "$(dirname ${BASH_SOURCE[0]})/s2i-common.sh"
 
 # s2i builder image name (fully qualified)
 declare -rx S2I_BUILDER_IMAGE="${S2I_BUILDER_IMAGE:-}"
+declare -rx E2E_IMAGE_SCRIPTS_URL="${E2E_S2I_IMAGE_SCRIPTS_URL:-}"
```

Comment:
Here you need to use the variable defined on `scripts/s2i-common.sh` instead of a end-to-end test variable.

Reply:
Ooo so this was the gravest error . Sure I will correct it.